compass/ocean/tests/overflow/hydro_vs_nonhydro/hydro_vs_nonhydro.cfg

```diff
@@ -0,0 +1,62 @@
+# config options for the horizontal grid
+[horizontal_grid]
+
+#Number of cells in the x-direction
+nx = 320
+
+#Number of cells in the y-direction
+ny = 4
+
+#Distance from two cell centers
+dc = 20.0
+
+# Options related to the vertical grid
+[vertical_grid]
+
+#Depth of the bottom of the ocean (m)
+bottom_depth = 200.0
+
+#Number of vertical levels
+vert_levels = 60
+
+# the type of vertical grid
+grid_type = uniform
+
+# The type of vertical coordinate (e.g. z-level, z-star)
+coord_type = z-star
+
+# Whether to use "partial" or "full", or "None" to not alter the topography
+partial_cell_type = None
+
+# config options for the solitary wave testcase
+[hydro_vs_nonhydro]
+
+#Lateral position of the shelf-break (m)
+xs = 4700.0
+
+#Length-scale of the slope (m)
+Ls = 533.3333
+
+#Beta in eos
+eos_linear_beta = 0.8
+
+#Reference salinity (PSU)
+eos_linear_Sref = 35.0
+
+#Reference density
+eos_linear_densityref = 1000.0
+
+#Density variation per layer
+rhoz = -2.0e-4
+
+#Lower temperature (deg C)
+lower_temperature = 10.0
+
+#Higher temperature (deg C)
+higher_temperature = 20.0
+
+# config options for the solitary wave testcase
+[visualize]
+
+#Time at which plotting the variables
```

Comment:
```suggestion
# Time index at which to plot the variables
```
This might be more convenient if written in hours so that the user doesn't have to check the streams file to see how frequently variables are output.

Reply:
Modified comment.